_inc/client/components/masthead/style.scss

```diff
@@ -68,3 +68,8 @@
 		text-align: left;
 	}
 }
+
+#sandbox-domain-badge {
+	background-color: red;
```

Comment:
What do you think about styling this a bit more like the sandbox badge on WordPress.com?

```suggestion
	background: #d54e21;
	text-transform: uppercase;
	letter-spacing: 0.2em;
	text-shadow: none;
	font-size: 9px;
	font-weight: bold;
	cursor: pointer;
```